docs/source/legacy-dev-docs.rst

```diff
@@ -151,13 +151,11 @@ system you can turn pool tracing on for an application with the following java f
 
 .. code-block:: bash
 
-    DECJ_MAXHEAP="-Dopendcs.connection.pool.trace=true" routsched ...
```

Comment:
this should be opendcs not cwms. I need to sanity check but this should've been renamed as part of a branch that was recently merged it.

(also I move the dev docs out of the legacy mark as they aren't, you'll want to rebase onto the most recent master branch)